src/app/components/Figure/Caption/index.test.jsx

```diff
@@ -5,6 +5,6 @@ import Caption from './index';
 describe('Caption', () => {
   shouldMatchSnapshot(
     'should render correctly',
-    <Caption>This is some Caption text</Caption>,
+    <Caption text="This is some Caption text" />,
```

Comment:
Are you sure you meant to make this change? Shouldn't the text be passed in as a child, rather than as a prop in this component?

Reply:
I think this got lost in merge conflicts, will address